tinygrad/renderer/cstyle.py

```diff
@@ -115,7 +115,8 @@ def ssa(prefix:str, u:Optional[UOp]=None):
       uop,dtype,src,args = u.op,u.dtype,u.src,u.arg
       # these four uops don't have output dtypes
       if uop is UOps.IF:
-        kk(f"if ({r[src[0]]}) {{")
+        try: kk(f"if ({r[src[0]]}) {{")
+        except KeyError: kk(f"if ({r[src[0].src[0]]}) {{")
```

Comment:
what is this?

Reply:
Unnecessary now, some key errors were occurring with the nested refs when I was making bigger changes